float64 xPos
float64 yPos
float64 area
float64 length
float64 distanceToCenterX
float64 distanceToCenterY
float64 distanceToBottomY
float64 depth
